website/pushnotifications/models.py

```diff
@@ -60,12 +59,6 @@ class Device(models.Model):
     date_created = models.DateTimeField(
         verbose_name=_("registration date"), auto_now_add=True, null=False
     )
-    language = models.CharField(
```

Comment:
Do we really want to remove this?

Reply:
Gladly, hasn't been used (or set properly) for years